app/controllers/request-to-go-live/organisation-address/post.controller.test.js

```diff
@@ -590,7 +593,8 @@ describe('organisation address post controller', () => {
       const mockServiceService = { updateService: mockUpdateService }
       const controller = getController(mockServiceService)
 
-      it('should update connector flag, update Stripe, log message then redirect to `Stripe > Add PSP account details` redirect', async function () {
+      it('should get the stripe account id directly from the account, update connector flag, ' +
```

Comment:
Similarly, could simplify description and if there are multiple and different things being tested, could split into different tests

Reply:
Please see above